src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaControllerTest.php

```diff
@@ -341,11 +347,10 @@ protected function setUpCollection()
         $this->collection->setStyle(json_encode($style));
 
         // Create Collection Type
-        $this->collectionType = new CollectionType();
-        $this->collectionType->setName('Default Collection Type');
-        $this->collectionType->setDescription('Default Collection Type');
+        $collectionTypeFixtures = new LoadCollectionTypes();
```

Comment:
How many collection types does this create? What if just one would be enough for this test? Then this test is much slower now, isn't it?

Reply:
It create the 2 types (Default and System) some test failed now because the system collection was not there or had a false id 🤔